pkg/api/v20230701preview/openshiftcluster.go

```diff
@@ -126,6 +128,7 @@ type ServicePrincipalProfile struct {
 	ClientSecret string `json:"clientSecret,omitempty" mutable:"true"`
 }
 
+// The outbound routing strategy used to provide your cluster egress to the internet.
```

Comment:
Should be set in `pkg/api/openshiftcluster.go` as well.

Reply:
In /pkg/api/openshiftcluster.go we have the following:

// OutboundType represents the type of routing a cluster is using
type OutboundType string

We can change it but technically this is new in the 07 01 preview API.